### PR TITLE
Add next version for che-machine-exec.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### Referenced issue:
https://github.com/eclipse/che/issues/13970

### What does this PR do?
Add next version for che-machine-exec. This version with activated security endpoint.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: v3/plugins/eclipse/che-machine-exec-plugin/next/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+publisher: eclipse
+name: che-machine-exec-plugin
+version: 0.0.1
```
Comment (skabashnyuk) [inline v3/plugins/eclipse/che-machine-exec-plugin/next/meta.yaml]: is it 0.0.1 or next?

Comment (AndrienkoAleksandr) [reply v3/plugins/eclipse/che-machine-exec-plugin/next/meta.yaml]: Thanks, fixed.

**Comment by l0rd** [comment]
@AndrienkoAleksandr are you going to update https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L665 as well to point to version `next`? 

And when we are release Che we need to release `che-machine-exec` as well:
- use a tag for the container image and on the git repository  (`7.0.0`)
- add the che-machine-exec-plugin version `7.0.0.` (in the `che-plugin-registry) 
- update `che.properties` to use che-machine-exec-plugin version `7.0.0`

@AndrienkoAleksandr @vparfonov are you ok with this approach?

File: v3/plugins/eclipse/che-machine-exec-plugin/next/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        protocol: ws
+        type: terminal
+        discoverable: false
+        secure: true
+        cookiesAuthEnabled: true
+  containers:
+   - name: che-machine-exec
+     image: eclipse/che-machine-exec
+     ports:
+       - exposedPort: 4444
```
Comment (l0rd) [inline v3/plugins/eclipse/che-machine-exec-plugin/next/meta.yaml]: Looks like you are missing a newline here

Comment (AndrienkoAleksandr) [reply v3/plugins/eclipse/che-machine-exec-plugin/next/meta.yaml]: Thanks, fixed.

**Comment by AndrienkoAleksandr** [comment]
> @AndrienkoAleksandr are you going to update https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L665 as well to point to version next?

After merge this pr che.properties will update @vparfonov 

>And when we are release Che we need to release che-machine-exec as well:
use a tag for the container image and on the git repository (7.0.0)
add the che-machine-exec-plugin version 7.0.0. (in the `che-plugin-registry)
update che.properties to use che-machine-exec-plugin version 7.0.0

+1 I am OK with this approach.

**Comment by AndrienkoAleksandr** [comment]
@l0rd Are you Ok if I merge this pr?

**Pull request merged**